Source/ThirdParty/AngelScript/source/as_config.h

```diff
@@ -178,6 +178,14 @@
 // Oracle Solaris Studio (previously known as Sun CC compiler)
 // __SUNPRO_CC is defined
 
+// Local (or Little) C Compiler
```

Comment:
Additionally, newly local modifications for Urho3D must be clearly marked with a code comment, e.g.  // URHO3D - reason for modification